pkg/commands/logging/s3/create.go

```diff
@@ -159,6 +161,10 @@ func (c *CreateCommand) ConstructInput(serviceID string, serviceVersion int) (*f
 		input.Domain = &c.Domain.Value
 	}
 
+	if c.FileMaxBytes.WasSet {
```

Comment:
As mentioned in the related Terraform PR: is this safe to send `0` to the API?

If yes, then nothing more to do here.

If no, then we could either add `&& c.FileMaxBytes.Value > 0` to the condition, or check the value is == 0 within the if block and return an error notifying the user that the input was unexpected.